spec.html

```diff
@@ -20782,7 +20782,7 @@
                 1. Assert: _module_ imports a specific binding for this export.
                 1. Let _importedModule_ be ? HostResolveImportedModule(_module_, _e_.[[ModuleRequest]]).
                 1. Let _indirectResolution_ be ? _importedModule_.ResolveExport(_e_.[[ImportName]], _resolveSet_, _exportStarSet_).
-                1. If _indirectResolution_ is not *null*, return _indirectResolution_.
+                1. Return _indirectResolution_.
```

Comment:
I'm not sure about the `Assert` above, but I do agree with the assessment of the problem in general. As for the change, probably just calling `1. Return ? _importedModule_.ResolveExport(...)`, not need to define `indirectResolution`.